lib/tasks/clear_vocab_sheets.rake

```diff
@@ -0,0 +1,5 @@
+namespace :vocab_sheets do
+  task clear: :environment do
+    VocabSheet.clear_vocab_sheet
```

Comment:
the name doesn't quite capture what it does.
maybe `purge_old_sheets!` <-- :elephant:

Reply:
aha.. Elspeth said the same thing :-D